### ISSUE TITLE
Shoutcast is not working, without further information

### ISSUE THREAD

**Issue by mbarsocchi** [rafael2k/darkice#172]
I'm trying to connect to a **shoutcast server** from a **darkice** client using Ubuntu. This is my configuration:

    #this section describes general aspects of the live streaming session
    [general]
    duration        = 0        # duration of encoding, in seconds. 0 means forever
    bufferSecs      = 10          # size of internal slip buffer, in seconds
    reconnect       = yes       # reconnect to the server(s) if disconnected
    realtime        = no       # run the encoder with POSIX realtime priority
    rtprio          = 3         # scheduling priority for the realtime threads
    
    # this section describes the audio input that will be streamed
    [input]
    device           = hw:CARD=PCH,DEV=0
    sampleRate      = 44100                 # sample rate in Hz. try 11025, 22050 or 44100
    bitsPerSample   = 16                    # bits per sample. try 16
    channel         = 2                     # channels. 1 = mono, 2 = stereo
    
    # this section describes a streaming connection to an IceCast2 server
    # there may be up to 8 of these sections, named [icecast2-0] ... [icecast2-7]
    # these can be mixed with [icecast-x] and [shoutcast-x] sections
    [shoutcast-0]
    bitrateMode     = cbr
    format          = mp3
    bitrate         = 96
    quality         = 1.0
    server          = xxxxxxxxxxxxxxx
    port            = 8020
    password        = xxxxxxxxxxxxxxx
    name            = Radio website
    url             = https://www.mywebsite.it
    genre           = live
    public          = no

But when I run

    darkice -v 10 -c /etc/darkice-shoutcast.cfg

It only shows this, without errors or similar, but there is no streaming at the url. Using **BUTT** it works. I've also tested with 8021 instead of 8020 for port (8020 it's the port number given by the provider) but no luck.

    DarkIce 1.4 live audio streamer, http://code.google.com/p/darkice/
    Copyright (c) 2000-2007, Tyrell Hungary, http://tyrell.hu/
    Copyright (c) 2008-2013, Akos Maroy and Rafael Diniz
    This is free software, and you are welcome to redistribute it 
    under the terms of The GNU General Public License version 3 or
    any later version.
    
    Using config file: /etc/darkice-shoutcast.cfg
    18-May-2021 12:02:28 Using ALSA DSP input device: hw:CARD=PCH,DEV=0
    18-May-2021 12:02:28 buffer size:  1764000
    18-May-2021 12:02:28 encoding
    18-May-2021 12:02:28 MultiThreadedConnector :: transfer, bytes 0
    18-May-2021 12:02:28 MultiThreadedConnector :: ThreadData :: threadFunction, was (thread, priority, type):  0x5568a502c010 0 SCHED_OTHER
    18-May-2021 12:02:28 MultiThreadedConnector :: ThreadData :: threadFunction, now is (thread, priority, type):  0x5568a502c010 0 SCHED_OTHER

**Comment by mbarsocchi** [comment]
**DEBUGGED**
It seems the error is related to the password and wrong parsing of the config file, so I've written it without spaces.
```
[shoutcast-0]
bitrateMode     = cbr
format          = mp3
bitrate         = 96
quality         = 1.0
server          = xxxxxxxxxxxxxxx
port            = 8020
password=xxxxxxxxxxxxxxx
name            = Radio website
url             = https://www.mywebsite.it
genre           = live
public          = no
```

**Comment by tvm1** [comment]
Indeed. Recent upgrade from 1.3 to 1.4 exposed this problem, as the previous config was working just fine.

1.4 will work only without any spaces. So, `password=foo`
